core/sequencer/server.go

```diff
@@ -313,14 +315,22 @@ func (s *Server) ApplyRevision(ctx context.Context, in *spb.ApplyRevisionRequest
 		return nil, err
 	}
 
-	// Apply mutations to values.
-	newLeaves, err := runner.ApplyMutations(entry.ReduceFn, indexedValues, leaves,
-		func(err error) { glog.Warning(err); mutationFailures.Inc(err.Error()) },
-	)
+	// Map Map Leaves
```

Comment:
This comment doesn't convey new information, compared to the function name below. Please remove it, or make it more specific, e.g. `// Convert Trillian map leaves into indexed KT updates.`